src/OpenLoco/Date.cpp

```diff
@@ -16,11 +16,21 @@ namespace OpenLoco
 
     static std::pair<month_id, uint8_t> getMonthDay(int32_t dayOfYear);
 
+    bool isLeapYear(const int year)
+    {
+        return year % 4 == 0;
```

Comment:
The original game is using this logic, but it's not _quite_ correct. According to Wikipedia:
> if (year is not divisible by 4) then (it is a common year)
> else if (year is not divisible by 100) then (it is a leap year)
> else if (year is not divisible by 400) then (it is a common year)
> else (it is a leap year)

Reply:
Perhaps too early to change it atm.

Reply:
Fair enough. Let's leave it as-is, then.